diff -ruN orig/oclint-0.13.1/oclint-core/cmake/OCLintConfig.cmake fixed/oclint-0.13.1/oclint-core/cmake/OCLintConfig.cmake
--- orig/oclint-0.13.1/oclint-core/cmake/OCLintConfig.cmake	2018-02-08 01:55:37.000000000 +0200
+++ fixed/oclint-0.13.1/oclint-core/cmake/OCLintConfig.cmake	2018-07-12 01:12:43.538279015 +0300
@@ -8,7 +8,7 @@
     IF(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
         SET(CMAKE_CXX_FLAGS "-fcolor-diagnostics")
     ENDIF()
-    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_LINKER_FLAGS} -fno-rtti -Wno-c++11-extensions -fPIC ${CMAKE_CXX_FLAGS}")
+    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_LINKER_FLAGS} -fno-rtti -fPIC ${CMAKE_CXX_FLAGS}")
 ENDIF()
 SET(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_CXX_LINKER_FLAGS} -fno-rtti")
 
diff -ruN orig/oclint-0.13.1/oclint-driver/main.cpp fixed/oclint-0.13.1/oclint-driver/main.cpp
--- orig/oclint-0.13.1/oclint-driver/main.cpp	2018-02-08 01:55:37.000000000 +0200
+++ fixed/oclint-0.13.1/oclint-driver/main.cpp	2018-07-12 01:13:05.695398710 +0300
@@ -178,7 +178,7 @@
 
 int main(int argc, const char **argv)
 {
-    llvm::cl::AddExtraVersionPrinter(&oclintVersionPrinter);
+    //llvm::cl::AddExtraVersionPrinter(&oclintVersionPrinter);
     CommonOptionsParser optionsParser(argc, argv, OCLintOptionCategory);
     oclint::option::process(argv[0]);
 
